src/09_dictionaries.py

```diff
@@ -34,11 +34,23 @@
 ]
 
 # Add a new waypoint to the list
-# YOUR CODE HERE
+new_waypoint = {
+    "lat": 56,
+    "lon": -136,
+    "name": "fourth place"
+}
+
+waypoints.append(new_waypoint)
+print(waypoints[-1])
 
 # Modify the dictionary with name "a place" such that its longitude
 # value is -130 and change its name to "not a real place"
-# YOUR CODE HERE
+waypoints[0]["lon"] = -130
+waypoints[0]["name"] = "not a real place"
```

Comment:
in the future, it's good to avoid hardcoding array index with magic numbers. Instead, it might be a good idea to change the values based on a condition being true...
```
for waypoint in waypoints:
    if (waypoint["name"] == "a place"):
        waypoint.update({"lon": -130, "name": "not a real place"})
        break
```